src/main/java/com/google/sps/servlets/RepresentativeListServlet.java

```diff
@@ -44,7 +44,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         HttpClient httpclient = HttpClients.createDefault();
         URIBuilder builder = new URIBuilder();
         URI uri = null;
-
+        
```

Comment:
there don't need to be changes in this file

Reply:
gonna remove this file from my pr.